testapps/ClassLibraryWithPrecompiledViews/ClassLibraryWithPrecompiledViews.csproj

```diff
@@ -12,6 +12,4 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="$(AspNetCoreVersion)" />
   </ItemGroup>
 
-  <Target Name="PrecompileRazorViews" AfterTargets="Build" DependsOnTargets="MvcRazorPrecompile" Condition="'$(TargetFramework)'!=''" />
```

Comment:
This fixes https://github.com/aspnet/MvcPrecompilation/issues/117. We explicitly call precompilation on the project from the consuming project.